include/vapor/FlowParams.h

```diff
@@ -97,32 +156,48 @@ class PARAMS_API FlowParams : public RenderParams {
+    //! \retval std::vector<long> - Number of seeds distributed on the X, Y, and Z axes.
     std::vector<long> GetGridNumOfSeeds() const;
-    void              SetGridNumOfSeeds(const std::vector<long> &);
+    
+    //! Sets the number of seed points on the X, Y, and Z axes if the seeding distribution is Gridded, as determined by GetSeedGenMode() (0 = Gridded, 1 = Random, 2 = Random with bias, 3 = List of seeds)
+    //! \retval std::vector<long> - Number of seeds distributed on the X, Y, and Z axes.
+    void SetGridNumOfSeeds(const std::vector<long> &);
 
-    /*
-     * 3 or 2 values to represent the number of seeds inside of a rake
-     * in the gridded seed generation mode.
-     */
+    //! Returns the number of seed points randomly generated if the seeding distribution is randomly generated, as determined by GetSeedGenMode() (0 = Gridded, 1 = Random, 2 = Random with bias, 3 = List of seeds)
+    //! \retval long - Number of seeds randomly distributed within the seeding rake region.
     long GetRandomNumOfSeeds() const;
+    
+    //! Setsthe number of seed points randomly generated if the seeding distribution is randomly generated, as determined by GetSeedGenMode() (0 = Gridded, 1 = Random, 2 = Random with bias, 3 = List of seeds)
+    //! \param[in] long - Number of seeds randomly distributed within the seeding rake region.
     void SetRandomNumOfSeeds(long);
 
-    /*
-     * The number of seeds in the random seed generation mode.
-     */
+    //! Returns the bias variable that randomly seeded flow-lines are distributed towards.
+    //! \retval string - The variable that seeds are biased distributed for.
     std::string GetRakeBiasVariable() const;
-    void        SetRakeBiasVariable(const std::string &);
+    
+    //! Sets the bias variable that randomly seeded flow-lines are distributed towards.
+    //! \retval string - The variable that seeds are biased distributed for.
+    void SetRakeBiasVariable(const std::string &);
 
+    //! When randomly seeding flowlines with bias towards a along a chosen variable's distribution, this returns the bias strength.  Negative bias will place seeds at locations where the bias value has low values.  Positive bias will place seeds where the bias variable has high values.
+    //! \retval int - The bias of the seed distribution.
     long GetRakeBiasStrength() const;
+
+    //! When randomly seeding flowlines with bias towards a along a chosen variable's distribution, this sets the bias strength.  Negative bias will place seeds at locations where the bias value has low values.  Positive bias will place seeds where the bias variable has high values.
+    //! \param[in] long - The bias of the seed distribution.
     void SetRakeBiasStrength(long);
 
+
     int  GetPastNumOfTimeSteps() const;
     void SetPastNumOfTimeSteps(int);
 
+    //! Returns the interval that new pathlines are injected into the scene.
+    //! \retval int - The seed injection interval.
```

Comment:
What is the unit here,  Integration steps, vapor time steps?

Reply:
I've added more explanation.